services/src/pro/permissions/mod.rs

```diff
@@ -110,6 +112,12 @@ impl From<DatasetId> for ResourceId {
     }
 }
 
+impl From<ModelId> for ResourceId {
```

Comment:
I think we'll leave permissions on models for later, so you can remove this here for now.

Reply:
4318dcc8

--- 

Commit MSG:

refactor: remove from<MlModelId> impl from ResourceId